docs/en/modules/admin/pages/features/notifications.adoc

```diff
@@ -15,3 +17,112 @@ image:features/notifications/notifications.png[Example of notifications page whe
+
+[options="header"]
+|============================================================================================================================================================================
+| 0.28 version                             |                                                                                       | Roles    |               |
+| Component                                | Action                                                                                | Follower | Affected user | Administrator
+| Admin                                    | A new attachment has been created                                                     | ✅     | ❌          | ❌
+|                                          | An admin requested an export                                                          | ❌     | ❌          | ✅
+|                                          | A component has been published in a participatory space                               | ✅     | ❌          | ❌
+| Verifications                            | A participant has tried to verify themself with the data of another participant       | ❌     | ❌          | ✅
+| Moderation                               | An administrator moderated a resource because it has been reported                    | ❌     | ✅          | ❌
+|                                          | A resource has been reported                                                          | ❌     | ❌          | ✅
+| User                                     | A user confirms the registration (if welcome notification is enabled)                 | ❌     | ✅          | ❌
+|                                          | A user earned a badge (if badges are enabled)                                         | ❌     | ✅          | ❌
+|                                          | A user has reached a new badge level (if badges are enabled)                          | ❌     | ✅          | ❌
+|                                          | A user has been officialized                                                          | ❌     | ❌          | ✅
+|                                          | A user asked her data export                                                          | ❌     | ✅          | ❌
+|                                          | A user I follow endorsed an ressource                                                 | ✅     | ❌          | ❌
+|                                          | A public profile has been updated                                                     | ✅     | ❌          | ❌
+| User groups (if user groups are enabled) | A new user group has been created                                                     | ❌     | ❌          | ✅
+|                                          | A user group with the name has updated its profile                                    | ❌     | ❌          | ✅
```

Comment:
This reads a little strange, I'm not sure what you mean by "user group with the name has updated" - with what name?

Reply:
Good catch, it's an error! 
I think I took the original translation and delete the part I didn't want but forgot the "with the name" part 😄 
The translation for this email: 
`A user group with the name %{user_group_name} has updated its profile, leaving it unverified. You can now verify it in the <a href="%{groups_admin_url}">admin panel</a>.`